WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerFragment.kt

```diff
@@ -97,7 +97,8 @@ class HomePagePickerFragment : Fragment() {
             layoutsRecyclerView.setVisible(!uiState.loadingIndicatorVisible && !uiState.errorViewVisible)
             AniUtils.animateBottomBar(bottomToolbar, uiState.isToolbarVisible)
             when (uiState) {
-                is UiState.Loading -> {}
+                is UiState.Loading -> { // Nothing more to do here
```

Comment:
Removing the `is UiState.Loading` case makes the `when` clause non exhaustive and produces a warning.
I added the comment to make it more clear that this case is already handled in the generic code above `when` but probably made it more confusing 😕

Reply:
Oh right! That makes sense.. for some reason I didn't see the warning (when I tried removing it) but thought it only mattered when omitting the default case for a when expression rather than statement. In any case, thanks for explaining! :+1: